.github/workflows/build.yml

```diff
@@ -199,6 +199,12 @@ jobs:
           exit 1
         fi
 
+        # Check that commit contains ':'
+        if git log --format=%s origin/"$BASE"..HEAD |  grep -v ':'; then
+          echo "Found a bad commit message"
+          exit 1
+        fi
+
```

Comment:
Maybe adapt the name of the CI step (currently "Check for 'fixup' commits), or create a different steps, it might make it easier to understand what's going on with separate steps (and names).

Reply:
done. I changed the script to output what's wrong on stdout